### PR TITLE
Change #dropzone-large ID to #comment-form-body

### PR DESCRIPTION
**Pull request by noi5e**
Small fix, this is fixing this inconsistency mentioned [here in this comment](https://github.com/publiclab/plots2/pull/9067#discussion_r564846813)

105936040-dc2e6e00-6007-11eb-8b42-6c9eccd138ea

In the past, when we hit the Preview button on a comment form, the editor toggled on B in the above image, which had the class `.dropzone`. This is a little difficult to read in the code (as mentioned in the comment linked above). B being a `.dropzone` has nothing to do with us toggling it for preview. So I changed the ID to `#comment-form-body-reply-123` (used to be `#dropzone-large-reply-123`), and toggle on that instead.

Also changed the ID to `#comment-form-body` in the edit form partial (used to be `#c123div`), and updated system tests.

---
(This issue is part of the larger Comment Editor Overhaul Project with Outreachy. Refer to Planning Issue #9069 for more context)

### CODE REVIEW

**Comment by noi5e** [comment]
Tests are passing, but hold off on merging this one for now!

I'm going to write a system test for toggling preview.

Also, this PR has a bunch of prerequisites! Visit my planning issue at #9069 and merge the PRs from top to bottom please!

File: app/assets/javascripts/editorToolbar.js [hunk 1]
```diff
@@ -69,12 +69,12 @@ $(function() {
           console.log($("#image-upload-progress-container-" + $E.commentFormID));
           $("#image-upload-progress-container-" + $E.commentFormID).show();
           $("#image-upload-text-" + $E.commentFormID).show();
-          $("#dropzone-choose-one-" + $E.commentFormID).hide();
+          $("#choose-one-" + $E.commentFormID).hide();
```
Comment (noi5e) [inline app/assets/javascripts/editorToolbar.js:72]: removing the dropzone part in the ID slug. choose one is already nested in another, larger dropzone, which I forgot, so this isn't necessary.

**Review by cesswairimu — APPROVED** [review]
Awesome :tada: :tada:

**Review by jywarren — APPROVED** [review]

**Comment by jywarren** [comment]
Awesome! Thanks both of you!!!!

**Pull request merged**